packages/components/link/__tests__/link.tests.tsx

```diff
@@ -541,7 +542,7 @@ describe("Link prefetching", () => {
     act(() => {
       render(
         <Provider value={storeInViewMode}>
-          {links.map((link, i) => (
+          {links.map((link) => (
             <Link key={link} link={link} className={"my-link-${i}"}>
```

Comment:
I noticed you removed the `i` variable; I believe `className` is broken now (i.e `my-link-undefined`)

Reply:
Thanks @nicholasio. I think this showed up a TS warning originally because the `"my-link-${i}"` was not a template string, so the `i` was not actually use anywhere 😅  Fixed now 🙂

Reply:
oh yeah, that makes sense!

Reply:
Ups, absolutely my fault. Eslint complained that it was not used and I deleted it assuming Eslint was right. Sorry about that, and nice catch!

Thanks for the fix @michalczaplinski 👍